src/ranged.cpp

```diff
@@ -161,19 +161,79 @@ bool player::handle_gun_damage( item &it, int shots_fired )
 {
     if( !it.is_gun() ) {
         debugmsg( "Tried to handle_gun_damage of a non-gun %s", it.tname() );
+        shots_fired = shots_fired + 0; // left this here in case someone needs shots_fired in the future.
```

Comment:
This statement has no effect and makes no sense whatsoever.

Reply:
It's to retain the int shots_fired code in the bool above. Otherwise GitHub says it's unused.

Reply:
So if it's unused, remove it. If anybody needs it, they can add it back in. They can even look up how it used to be via the git history. There is no point in keeping dead code around.